frontend/src/api/Api.js

```diff
@@ -1,9 +1,18 @@
 import axios from 'axios';
 
-const baseUri = 'http://localhost:61346/api';
+let baseUri;
+if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') {
+  baseUri = 'http://localhost:61346/api';
+} else {
+  baseUri = 'http://13.81.9.144/api';
+}
 
 const Api = axios.create({
   baseURL: baseUri,
+  headers: {
+    'Cache-control': 'no-cache',
+    Pragma: 'no-cache',
+  },
```

Comment:
Why this was added/changed?

Reply:
When history.goBack() is done the browser returns cached request to the frontend. That's why you saw videos that you had already deleted in Recycling Bin when you went to the profile page and back. 

This here tells the browser to not cache any requests